Update insights related documentation (#541)
wso2/docs-choreo-dev: en/docs/insights/view-api-insights.md

  ![Total traffic](../assets/img/insights/overview-page-total-traffic.png){.cInlineImage-half}

>   This widget displays the total traffic your selected environment received during the selected time range. This includes both successful requests and error requests. To investigate further, you can click the arrow icon on the bottom right corner of the widget to open the [**Traffic Page**](#traffic).

Reviewer: Why was **Traffic** page changed to **Traffic Page**?

Author: Isn't it bit odd to make page  'p' in simple and Traffic 'T' in capital? And I guess we need to bold Traffic Page as whole and not only Traffic.

  ![Average error rate](../assets/img/insights/overview-page-error-rate.png){.cInlineImage-half}

>   This widget displays the average error rate (error count/total request count) in your selected environment for the selected time interval. You can use this widget as an indicator to know the health of the system. If the error rate is high, you can investigate further by clicking the arrow on the bottom right of the widget and open the [**Errors Page**](#errors).

Reviewer: ```suggestion
This widget displays the average error rate (i.e., error count/total request count) of the selected environment for a given time interval. You can use this widget as an indicator to understand the health of the system. If the error rate is high, you can investigate further by clicking the arrow on the bottom right of the widget and opening the [**Errors Page**](#errors).
```

- **95th Percentile Latency**

  ![95th percentile latency](../assets/img/insights/overview-page-latency.png){.cInlineImage-half}

>   This widget displays the 95th percentile of all API latencies in your selected environment for the selected time interval. You can use this widget to know whether the whole system operates under given SLAs. This metric gives the first indication of slow APIs. To investigate further, you can click the arrow on the bottom right of the widget to open the [**Latency Page**](#latency) where you can further analyze the latency.

Reviewer: ```suggestion
This widget displays the 95th percentile of all API latencies in your selected environment for the given time interval. You can use this widget to know whether the complete system operates under given SLAs. This metric provides the first indication of slow APIs. To investigate further, you can click the arrow on the bottom right of the widget to open the **Latency Page**](#latency) where you can further analyze the latency.
```

  ![API usage by application](../assets/img/insights/usage-by-application.png){.cInlineImage-half}

- **API Usage By Target**

>   This widget shows the per-backend breakdown of requests for the APIs you selected. This is useful when multiple APIs are sharing the same backend, which has traffic restrictions. You can use these stats to proactively scale your backends. You can switch between the two views using the small icon at the upper-right corner of the widget.

Reviewer: ```suggestion
This widget shows the per-backend breakdown of requests for the APIs you selected. This information is useful when multiple APIs share the same backend that has traffic restrictions. You can use these stats to scale your backends proactively. You can switch between the two views using the small icon at the upper-right corner of the widget.
```

  ![Errors by status code](../assets/img/insights/error-by-status-code.png){.cInlineImage-full}

- **Target Errors by Status Code**

>   This graph shows the distribution of HTTP status codes received for target errors over time for the selected period. Apply the required filters as explained above to select the APIs and status code groups to which this content applies. When you select multiple APIs, the target error count is grouped by the status code.

Reviewer: ```suggestion
This graph shows the distribution of target errors that have occurred during the selected time interval by HTTP status code. Apply the required filters as explained in the table above to view the APIs and status code groups to which this content applies. If you select multiple APIs, the widget groups the target error count by the status code.
```

  ![Target errors by status code](../assets/img/insights/target-error-by-status-code.png){.cInlineImage-full}

- **Errors**

>   Errors heat map shows the HTTP response codes of errors that are sent to the client by the API Gateway. Each status code is shown against the selected APIs. The cell color is determined by the number of errors belonging to each status code. The cell color is red for higher numbers and white for lower numbers. Use this chart to further drill down on error code groups that you identified with the **Errors by Status Code** graph.

Reviewer: ```suggestion
The Errors heat map shows the HTTP response codes of errors sent to the client by the API Gateway. Each row displays the number of times the system returned each status code for the given API/application. The cell color is red for higher numbers and white for lower numbers. Use this chart to further drill down on error code groups that you identified with the **Errors by Status Code** graph.
```

  ![Errors](../assets/img/insights/error-heatmap.png){.cInlineImage-full}

- **Target Errors**

>   Target errors heat map shows the HTTP response codes of errors that are received by the API Gateway from the backend. Each status code is shown against the selected APIs. The cell color is determined by the number of errors belonging to each status code. The cell color is red for higher numbers and white for lower numbers. Use this chart to further drill down on error code groups that you identified with the **Target Errors by Status Code** graph.

Reviewer: ```suggestion
The Target errors heat map shows the HTTP response codes of errors that the API Gateway received from the backend. Each row displays the number of times the system returned each status code for the given API/application. The cell color is red for higher numbers and white for lower numbers. If required, you can further filter the results by the error code groups you identified with the **Target Errors by Status Code** graph.
```

- **Cache Hit Percentage**

>   This graph shows the percentage of requests that were handled via the response cache overtime and the total hits over time. This allows you to assess the efficiency with which the API requests are handled. For example, if the cache hit rate is low, it may indicate that the backend is handling the request to return the same response multiple times instead of returning the response via the cache. In such a scenario, there is scope to improve performance via response caching.

Reviewer: ```suggestion
This graph shows the percentage of requests the system has handled via the response cache over time and the total hits over time. This information allows you to assess how efficiently the backend handles API requests. For example, if the cache hit rate is low, it may indicate that the backend generates the same response each time a specific request is sent instead of returning the response via the cache. In such a scenario, there is scope to improve performance via response caching.
```

  ![Cache hit percentage](../assets/img/insights/cache-statistics.png){.cInlineImage-full}

- **Latency**

>   This section shows the total latency that occurred at the same times corresponding to the above **Cache Hit Percentage** graph.

Reviewer: ```suggestion
This section shows the total latency reported during the same time interval applied to the **Cache Hit Percentage** graph above.
```
